index.js

```diff
@@ -3,28 +3,35 @@
 // Refactor the function so that it does not allow to enter an empty array (throw)
 // Test the refactored function
 
-const minVal = arrVal => Math.min(...arrVal);
+const minVal = arrVal => {
+    if(arrVal.length === 0){
+        throw new Error("Array cannot be empty");
+    }
+    return Math.min(...arrVal);
+}
 
 // Test function fizzBuzz thoroughly
 // Does it have a logic error?
 // Refactor and test some more
 
 const fizzBuzz = n => {
     let result = [];
-    for(let i=0;i<n;i++){
```

Comment:
I agree that this error should not be there.
However, the logic error was by design.